examples/reload/monitor-progress/monitor-progress.go

```diff
@@ -13,10 +16,17 @@ import (
 const testDataFolder = "/testdata"
```

Comment:
I assume we can remove this folder and its content now?

Reply:
I missed that, there was also a prerequisite section in the README.md that had to be removed